language-support/ts/daml-ledger/index.test.ts

```diff
@@ -57,48 +59,57 @@ afterEach(() => {
 describe("streamQuery", () => {
   // we only check that nothing crashes.
   test("receive unknown message", async () => {
+    const restoreConsole = mockConsole();
     const server = new WS("ws://localhost:4000/v1/stream/query", {
       jsonProtocol: true,
     });
     const ledger = new Ledger(mockOptions);
     const stream = ledger.streamQuery(Foo);
     await server.connected;
     console.log("connected");
```

Comment:
To which console does this go now? I suppose this helped you during writing the tests. Maybe we can just remove it now?

Reply:
no, I think this is a clean up function that is called at the end of the test block, so that other errors go back to the normal console. actually, it would be better to call this just around the `server.send` block.

Reply:
moved the mockConsole call around the server.send call.